release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1267,6 +1267,8 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-15-cloud-etcd-operator-bug-fixes"]
 ==== etcd Cluster Operator
 
+* Previously, the `cluster-backup.sh` script cached the `etcdctl` binary on the local machine indefinitely, making updates impossible. Now, the `cluster-backup.sh` script pulls the latest `etcdctl` binary each time it is run. (link:https://issues.redhat.com/browse/OCPBUGS-19052[*OSDOCS-19052*])
```

Comment:
Not sure if it's a definite rule, but suggestion to follow the typical wording I see elsewhere:
```suggestion
* Previously, the `cluster-backup.sh` script cached the `etcdctl` binary on the local machine indefinitely, making updates impossible. With this update, the `cluster-backup.sh` script pulls the latest `etcdctl` binary each time it is run. (link:https://issues.redhat.com/browse/OCPBUGS-19052[*OSDOCS-19052*])
```